crates/chia-consensus/src/spendbundle_validation.rs

```diff
@@ -46,32 +46,54 @@ pub fn validate_clvm_and_signature(
             .flat_map(move |(condition, items)| {
                 let spend_clone = spend.clone();
                 items.iter().map(move |(pk, msg)| {
```

Comment:
is there a good reason to *move* the spends into this lambda function? That's probably why you have to clone the spends in the first place.